File: Flutter/app_eventos/lib/core/models/dto/fight_dto.dart
class FightDto {
  final int fightId;
  final int tournamentId;
  final int fighter1Id;
  final int fighter2Id;
  final String? status;
  final int? winnerId;
  final String? nombrePeleador1;
  final String? nombrePeleador2;

  FightDto({
    required this.fightId,
    required this.tournamentId,
    required this.fighter1Id,
    required this.fighter2Id,
    this.status,
    this.winnerId,
    this.nombrePeleador1,
    this.nombrePeleador2,
  });

  factory FightDto.fromJson(Map<String, dynamic> json) => FightDto(
        fightId: json['fightId'],
        tournamentId: json['tournamentId'],
        fighter1Id: json['fighter1Id'],
        fighter2Id: json['fighter2Id'],
        status: json['status'],
        winnerId: json['winnerId'],
        nombrePeleador1: json['nombrePeleador1'],
        nombrePeleador2: json['nombrePeleador2'],
      );
}

File: TFC-AppEventos/TFC.AppEventos.Application.DTO/FightDTO.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TFC.AppEventos.Application.DTO
{
    public class FightDto
    {
        public int FightId { get; set; }
        public int TournamentId { get; set; }
        public int Fighter1Id { get; set; }
        public int Fighter2Id { get; set; }
        public string? Status { get; set; }
        public int? WinnerId { get; set; }
        public string? NombrePeleador1 { get; set; }
        public string? NombrePeleador2 { get; set; }
    }
}
